Makefile

```diff
@@ -55,3 +55,33 @@ coverage:
 coveralls: coverage
 	go get github.com/mattn/goveralls
 	goveralls -coverprofile=$(COVERAGE_FILE) -service=github
+
+DURATION ?= 3s
+COUNT ?= 5
+BENCH_PATH ?= bench
```

Comment:
I propose to use absolute paths instead of relative.
How to find a path to a directory where Makefile is located:
```
# This way everything works as expected ever for
# `make -C /path/to/project` or
# `make -f /path/to/project/Makefile`.
MAKEFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
PROJECT_DIR := $(patsubst %/,%,$(dir $(MAKEFILE_PATH)))
```

Sometimes people like to run Makefiles in other directories ("out of source" build) and using of absolute path will guarantee that you are always using exactly same directories.

Reply:
Fixed.